### PR TITLE
Feature/caes 368  fix envs

### PR DESCRIPTION
**Pull request by mg-4xxi**

### CODE REVIEW

File: .env.dist [hunk 1]
```diff
@@ -1,70 +1,70 @@
...
-SUPERADMIN_PASS='$2y$13$BNuHqfNls76Bsqmb8X8UfOTDWbpTwu0VTxPYw65Um0iDw.KPFYzNe'
-OAUTH_ALLOWED_DOMAINS=example.com,gmail.com
 
-## Replace with your UID(echo $UID)
-PARENT_USER_ID=1001
-# Docker configuration!
-SERVER_HTTP_PORT=80
+#### Access to admin panel
+##### Gen new pass - bin/console security:encode-password 'pass_here' 'Symfony\Component\Security\Core\User\User'
+SUPERADMIN_PASS='$2y$13$BNuHqfNls76Bsqmb8X8UfOTDWbpTwu0VTxPYw65Um0iDw.KPFYzNe'
```
Comment (atrutko-4xxi) [inline .env.dist]: @mg-4xxi @beshkenadze There is a suggestion to generate this variable at startup. Using Docker ARG

File: .env.dist [hunk 2]
```diff
@@ -1,70 +1,70 @@
...
 
-# Postgres credentials
+#### Data Base Configuration
```
Comment (atrutko-4xxi) [inline .env.dist]: @mg-4xxi What version of the database?

File: .env.dist [hunk 3]
```diff
@@ -1,70 +1,70 @@
...
 POSTGRES_DB=dev
 POSTGRES_USER=dev
 POSTGRES_PASSWORD=dev
-
-###> doctrine/doctrine-bundle ###
-# Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
-# For an SQLite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
-# Configure your db driver and server_version in config/packages/doctrine.yaml
 DATABASE_URL=pgsql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:5432/${POSTGRES_DB}
 TEST_DATABASE_URL=pgsql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db_test:5432/${POSTGRES_DB}
```
Comment (atrutko-4xxi) [inline .env.dist]: @mg-4xxi what is it?

File: .env.dist [hunk 4]
```diff
@@ -1,70 +1,70 @@
...
-###> symfony/framework-bundle ###
+#### App environment
+##### dev  - for development server
+##### prod - for production server
+##### test - for tests
 APP_ENV=dev
+##### Restriction by domains
+OAUTH_ALLOWED_DOMAINS=example.com,gmail.com
+##### Private signature to encrypt application data
 APP_SECRET=
```
Comment (atrutko-4xxi) [inline .env.dist]: @mg-4xxi @beshkenadze what are the secret requirements?

File: .env.dist [hunk 5]
```diff
@@ -1,70 +1,70 @@
...
+##### Secret hash to encrypt backup codes stored in application data base
 BACKUP_CODE_SALT=salt
```
Comment (atrutko-4xxi) [inline .env.dist]: what are the secret requirements?

File: .env.dist [hunk 6]
```diff
@@ -1,70 +1,70 @@
...
-BACKUP_CODE_HASH_LENGTH=10
+##### Secret hash to encrypt invitations stored in application data base
 INVITATION_SALT=salt
```
Comment (atrutko-4xxi) [inline .env.dist]: what are the secret requirements?

File: .env.dist [hunk 7]
```diff
@@ -1,70 +1,70 @@
...
-MAILER_URL=null://localhost
-DELIVERY_ADDRESS=your_dev@email.com
-APP_NAME='Caesar App'
-MAILER_PORT_1080=1080
-MAILER_PORT_1025=1025
-###< symfony/swiftmailer-bundle ###
-
-###> lexik/jwt-authentication-bundle ###
+##### Is a text used to control access to a jwt token
 JWT_PASSPHRASE=
```
Comment (atrutko-4xxi) [inline .env.dist]: what are the secret requirements?

File: .env.dist [hunk 8]
```diff
@@ -1,70 +1,70 @@
...
-###< lexik/jwt-authentication-bundle ###
 
-###> hwi/oauth-bundle ###
+#### Google Authentication
 GOOGLE_ID=id
```
Comment (atrutko-4xxi) [inline .env.dist]: What is the variable record format?

File: .env.dist [hunk 9]
```diff
@@ -1,70 +1,70 @@
...
-###< hwi/oauth-bundle ###
-
-#Setup your project path to open files by link from browser https://symfony.com/doc/current/reference/configuration/framework.html#ide
-#For Phpstorm use such string: phpstorm://open?url=file://%f&line=%l&/var/www/html/>/path/to/project/on/host/machine
-IDE_STRING=''
+##### Allowed urls to redirect by Google authentication
+ALLOW_FRONT_REDIRECT_PATTERN='/^http[s]*:\/\/(example.com|example2.com)/'
 
-###> snc/redis-bundle ###
+#### Redis
```
Comment (atrutko-4xxi) [inline .env.dist]: Will there be an indication of the database? 0-16

File: .env.dist [hunk 10]
```diff
@@ -1,70 +1,70 @@
...
 REDIS_HOST=redis
 REDIS_PORT=6379
```
Comment (atrutko-4xxi) [inline .env.dist]: Is the port indication mandatory?

File: .env.dist [hunk 11]
```diff
@@ -1,70 +1,70 @@
...
+#### Email System
+
+##### For Gmail as a transport, use: "gmail://username:password@localhost"
+##### For a generic SMTP server, use: "smtp://localhost:25?encryption=&auth_mode="
+##### Delivery is disabled by default via "null://localhost"
+MAILER_URL=null://localhost
+
+#### Docker
+##### Docker server port
+SERVER_HTTP_PORT=80
```
Comment (atrutko-4xxi) [inline .env.dist]: Is the port indication mandatory?

File: .env.dist (hunk 2 above)
Comment (mg-4xxi) [reply .env.dist]: the database version variable replaced to doctrine.yaml > parameters section

File: .env.dist (hunk 3 above)
Comment (mg-4xxi) [reply .env.dist]: doctrine settings for db connection
actually we can replace these to doctrine.yaml too

**Pull request closed without merging**